samples/scanner/lb.yaml

```diff
@@ -0,0 +1,32 @@
+rules:
```

Comment:
Maybe IP's in test rules can be picked from the subnets reserved in RFC 5737 for documentation purposes, i.e. `192.0.2.0/24`, `198.51.100.0/24` and `203.0.113.0/24`?